src/stdlib_experimental_error.md

```diff
@@ -0,0 +1,106 @@
+# Catching and handling errors
+
+* [`check` - Checks the value of a logical condition](#check---checks-the-value-of-a-logical-condition)
+* [`error_stop` - aborts a program](#error_stop---aborts-a-program)
```

Comment:
Should we say "Aborts _the_ program" here? There's only one program we could abort, so I think we should use _the_ instead of _a_. 

I'm not a native english speaker though, so not sure. @everythingfunctional what's the correct article to use here?

Reply:
I would say *the* would be more appropriate here.